src/pages/ValidateLoginPage/index.website.js

```diff
@@ -117,14 +117,14 @@ class ValidateLoginPage extends Component {
     }
 
     render() {
-        const isTfaRequired = lodashGet(this.props, 'account.requiresTwoFactorAuth', false);
+        const is2FARequired = lodashGet(this.props, 'account.requiresTwoFactorAuth', false);
         const isSignedIn = Boolean(lodashGet(this.props, 'session.authToken', null));
         return (
             <>
                 {this.getAutoAuthState() === CONST.AUTO_AUTH_STATE.FAILED && <ExpiredValidateCodeModal />}
```

Comment:
We have 5 calls in sequence to a method which will always give the same result during a `render`. Maybe we could extract `this.getAutoAuthState()` to a variable `const autoAuthState = this.getAutoAuthState();` and use it.